### PR TITLE
better flexbox example

### PR DESCRIPTION
**Pull request by NickIliev**

### CODE REVIEW

File: app/ui-category/layouts/flexbox-layout-three/article.md [hunk 1]
```diff
@@ -7,7 +7,42 @@ Direction also plays a role here, determining the direction new lines are stacke
...
+
+
+## Justify Content
+
+This defines the alignment along the main axis. It helps distribute extra free space left over when either all the flex items on a line are inflexible, 
+or are flexible but have reached their maximum size. It also exerts some control over the alignment of items when they overflow the line.
+
+## flex-end 
+
+Items are packed toward to end line
```
Comment (zh-m) [inline app/ui-category/layouts/flexbox-layout-three/article.md]: toward the end line

File: app/ui-category/layouts/flexbox-layout-three/article.md [hunk 2]
```diff
@@ -7,7 +7,42 @@ Direction also plays a role here, determining the direction new lines are stacke
...
 
+### wrap-reverse
+
+multi-line / right to left in ltr; left to right in rtl
+<snippet id='flexbox-wrap-reverse-html'/>
+
+
+## Justify Content
+
+This defines the alignment along the main axis. It helps distribute extra free space left over when either all the flex items on a line are inflexible,
```
Comment (zh-m) [inline app/ui-category/layouts/flexbox-layout-three/article.md]: helps to distribute

**Comment by vakrilov** [comment]
Much better!

**Comment by vakrilov** [comment]
meme

**Pull request merged**